Source/WebKit/NetworkProcess/storage/OriginQuotaManager.cpp

```diff
@@ -156,9 +158,9 @@ void OriginQuotaManager::resetQuotaForTesting()
     m_quotaCountdown = 0;
 }
 
-uint64_t OriginQuotaManager::reportedQuota() const
+uint64_t OriginQuotaManager::reportedQuota()
 {
-    if (m_usage && *m_usage > m_standardReportedQuota)
+    if (usage() > defaultReportingRealQuotaRatio * m_standardReportedQuota)
```

Comment:
The more conservative idea was that we would increase the reported quota if usage gets higher.
To get the actual quota (and so derive the disk size), the web application would need to write more than half of the standard initially reported quota.
How about something like increasing m_standardReportedQuota (say multiply it by 2) in this if branch?

Reply:
Do you mean change it to: `if (usage() * 2 >= m_standardReportedQuota)`?

Reply:
Change it to something like:
```
if (usage() > defaultReportingRealQuotaRatio * m_standardReportedQuota)
    m_standardReportedQuota *= 2; // or maybe std::min(m_quota, 2 * m_standardReportedQuota)
return std::min(m_quota, m_standardReportedQuota);
```